spec/decorators/service_provider_session_decorator_spec.rb

```diff
@@ -1,6 +1,9 @@
 require 'rails_helper'
 
 RSpec.describe ServiceProviderSessionDecorator do
+  subject { ServiceProviderSessionDecorator.new(sp_name: sp_name, sp_logo: nil) }
```

Comment:
Oh yeah I noticed that...I guess my thought was that if we were going to use `subject`, we should just call it `subject` in the tests? Otherwise might as well use a `let(:decorator)` statement, right?